doc/src/thermo_modify.rst

```diff
@@ -75,6 +76,34 @@ are drifting out of the box through a fixed boundary condition (see
 the :doc:`boundary <boundary>` command).  In this case one atom may be
 deleted before the rest of the molecule is, on a later timestep.
 
+The *warn* keyword allows you to control whether LAMMPS will print
+warning messages and how many of them.  Most warning messages are only
+printed by MPI rank 0.  They are usually pointing out important issues
+that should be investigated, but LAMMPS cannot determine for
+certain whether they are an indication of an error.
+
+Some warning messages are printed during a run (or immediately before)
+each time a specific MPI rank encounters the issue, e.g. bonds that are
+stretched too far or dihedrals in extreme configurations. These can
+become verbose for and thus quickly blow up the size of the log file
+and screen output.  Thus a limit of 100 warning messages is applied
+by default.  The warning count is applied to the entire input unless
+reset with a ``thermo_modify warn reset`` command.  If there are more
+warnings than the limit, LAMMPS will print one final warning that it
+will not print any additional warning messages.
+
```

Comment:
NOTE (in rst lingo) The limit is enforced on either a per-processor basis or summed count across processors which is only updated at thermodyanmic output steps.   Thus if running on 1000s of processors with rare thermodynamic output, many more warnings than the limit could potentially be output

Reply:
Done. I like this. This keeps the complex explanation optically separate from the simplified description.